cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1658,10 +1764,12 @@ pub(super) mod test {
 
     #[test]
     fn insert_adds_install_table_when_missing() {
-        let test_packages = vec![PackageToInstall::from_str("foo").unwrap()];
+        let test_packages = vec![PackageToInstall::Catalog(
+            CatalogPackage::from_str("foo").unwrap(),
+        )];
```

Comment:
nit: we could just derive `From` impls for `PackageToInstall` and call `into` here, thats how the lockfile types work at least.